gpii/node_modules/transformer/src/js/Transformer.js

```diff
@@ -25,7 +25,55 @@ fluid.registerNamespace("gpii.transformer");
 
 (function () {
     /**
-     * Helper function for `gpii.transformer.configurationToSettings`. Used to modify the settings handler
+     * Helper function to find out if the capabilitiesTransformations for a settingsHandler block has inputPaths
+     * to produce output for given setting. Return true if has, otherwise, return false. Setting paths that are
```

Comment:
I'd better reword this into "If so, return true, otherwise, return false". But hey, that's only my opinion.